python/python/Ice/ObjectPrx.py

```diff
@@ -21,13 +47,13 @@ def checkedCast(type, proxy, facet=None, context=None):
     Returns
     -------
     ObjectPrx|None
-        A proxy with the requested type, or None if the target object does not support the requested type.
+        A proxy with the requested type and facet, or None if the target object does not support the requested type.
```

Comment:
I'd rather leave facet out the Returns description.

Reply:
fixed.